jekyll/_cci2/orbs-best-practices.md

```diff
@@ -1,112 +1,226 @@
+{:.no_toc}
+
+Commands, Jobs, Executors, Examples, and Parameters can all accepts descriptions. Ensure each and every component of your orb has a helpful description and provides any additional documentation that may be needed.
+
+## Structure
+
+### @orb.yml
+
+The `@orb.yml` file acts as the "root" of our project and contains much of the meta-data for our orb which will appear on the Orb Registry as well as the CLI.
 
-- Must have at least 1 [usage example]({{ site.baseurl }}/2.0/orb-author/#providing-usage-examples-of-orbs).
-- Show orb version as `x.y` (patch version may not need to be included) in the example.
-- Example should include most common/simplest use case calling a top-level job or other base-case elements if no job is present.
-- If applicable, you may want to showcase the use of [pre and post steps]({{ site.baseurl }}/2.0/reusing-config/#using-pre-and-post-steps) in conjunction with an orb’s job. 
+#### ALL Orbs Should Have A Description
+{:.no_toc}
+
+When orbs are published to the Orb Registry they are searchable by their name and description. Besides giving your users a better idea of the purpose and functionality of your orb, good descriptions are important for search optimization.
+
+#### Include Display Links
+{:.no_toc}
+
+Orbs utilize a special config key "[display]({{site.baseurl}}/2.0/orb-author/#orbyml)" which can hold a `source_url` for linking to your Git repository which holds the orb source code, and `home_url` to link to the product or service home page if applicable.
+
+```yaml
+display:
+  home_url: "https://www.website.com/docs"
+  source_url: "https://www.github.com/EXAMPLE_ORG/EXAMPLE_PROJECT"
+```
 
 ### Commands
 
-- In general, all orbs should contain at least one command. 
-- Some exceptions may include creating an orb for the sole task of providing an executor.
-- Combine one or more parameterizable steps to simplify a task.
-- All commands available to the user should complete a full task. Do not create a command for the sole purpose of being a “partial” to another command unless it can be used on its own.
-- It is possible not all CLI commands need to be transformed into an orb command. Single line commands with no parameters do not necessarily need to have an orb command alias.
-- Command descriptions should call out any dependencies or assumptions, particularly if you intend for the command to run outside of a provided executor in your orb.
-- It is a good idea to check for the existence of required parameters, environment variables or other dependencies as a part of your command.
+#### Most Orbs Will Contain At Least One Command
+{:.no_toc}
 
-example:
+Most, but not all orbs will contain at least a single command. Commands are used to execute shell commands and special CircleCI steps automatically on the user's behalf. In less common situations, for instance if a tool _requires_ the use of a particular Docker container, an orb may not project commands and only provide jobs.
```

Comment:
unsure of use of "project" here - maybe a typo or I am not getting meaning

Reply:
s/project/provide/ it looks like. Although maybe "contain" is a nicer way of avoiding repetition.